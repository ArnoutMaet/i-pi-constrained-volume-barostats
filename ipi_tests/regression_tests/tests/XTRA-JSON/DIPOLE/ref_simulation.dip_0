 #*EXTRAS*# Step:           0  Bead:      0  
     3.37323197          -1.73507010          -0.69532246
 #*EXTRAS*# Step:           1  Bead:      0  
     3.37271307          -1.73494828          -0.69202365
 #*EXTRAS*# Step:           2  Bead:      0  
     3.37242741          -1.73487769          -0.69013233
 #*EXTRAS*# Step:           3  Bead:      0  
     3.37154970          -1.73464730          -0.68401026
 #*EXTRAS*# Step:           4  Bead:      0  
     3.37133340          -1.73454431          -0.68205561
 #*EXTRAS*# Step:           5  Bead:      0  
     3.37137320          -1.73454576          -0.68202567
 #*EXTRAS*# Step:           6  Bead:      0  
     3.37147000          -1.73454983          -0.68245559
 #*EXTRAS*# Step:           7  Bead:      0  
     3.37170518          -1.73454879          -0.68341885
 #*EXTRAS*# Step:           8  Bead:      0  
     3.37186962          -1.73454890          -0.68416299
 #*EXTRAS*# Step:           9  Bead:      0  
     3.37187168          -1.73455371          -0.68424475
 #*EXTRAS*# Step:          10  Bead:      0  
     3.37196312          -1.73457104          -0.68482650
