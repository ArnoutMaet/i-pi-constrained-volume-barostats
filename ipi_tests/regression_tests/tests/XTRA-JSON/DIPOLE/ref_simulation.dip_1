 #*EXTRAS*# Step:           0  Bead:      1  
     2.98884114          -1.71803617          -0.77779596
 #*EXTRAS*# Step:           1  Bead:      1  
     3.00329481          -1.75041835          -0.71211260
 #*EXTRAS*# Step:           2  Bead:      1  
     3.01950183          -1.74684187          -0.70921815
 #*EXTRAS*# Step:           3  Bead:      1  
     3.11629817          -1.73107287          -0.70950363
 #*EXTRAS*# Step:           4  Bead:      1  
     3.18681304          -1.73053170          -0.70720831
 #*EXTRAS*# Step:           5  Bead:      1  
     3.19034924          -1.72063875          -0.70489030
 #*EXTRAS*# Step:           6  Bead:      1  
     3.18817026          -1.71903330          -0.70531992
 #*EXTRAS*# Step:           7  Bead:      1  
     3.20550744          -1.71387713          -0.69785462
 #*EXTRAS*# Step:           8  Bead:      1  
     3.23015378          -1.71793367          -0.68780763
 #*EXTRAS*# Step:           9  Bead:      1  
     3.23447938          -1.71647527          -0.68843501
 #*EXTRAS*# Step:          10  Bead:      1  
     3.25279956          -1.71641728          -0.68326590
