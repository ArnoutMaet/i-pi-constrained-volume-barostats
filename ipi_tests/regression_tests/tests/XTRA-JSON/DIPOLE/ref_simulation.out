# column   1     --> step : The current simulation time step.
# column   2     --> potential{electronvolt} : The physical system potential energy.
    0.00000000e+00     1.18368288e-01   
    1.00000000e+00     4.25536860e-02   
    2.00000000e+00     3.14110661e-02   
    3.00000000e+00     1.10247042e-02   
    4.00000000e+00     7.61768008e-03   
    5.00000000e+00     6.17034118e-03   
    6.00000000e+00     5.17160877e-03   
    7.00000000e+00     3.63951782e-03   
    8.00000000e+00     3.40286133e-03   
    9.00000000e+00     2.50063956e-03   
    1.00000000e+01     1.87091865e-03   
